packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTable.tsx

```diff
@@ -62,6 +70,73 @@ export const TransactionDateTime = ({
   )
 }
 
+export const CustomAddressTxExplorer = ({
+  tx,
+  explorerClassName = 'arb-hover underline'
+}: {
+  tx: MergedTransaction
+  explorerClassName?: string
+}) => {
+  const { address } = useAccount()
+  const { l1, l2 } = useNetworksAndSigners()
+
+  const isCustomSenderTx = useMemo(() => {
```

Comment:
Somehow I think this may be more intuitive, as you don't really specify it yourself, so it's not really custom
```suggestion
  const isDifferentSenderTx = useMemo(() => {
```